README.md

```diff
@@ -266,10 +267,11 @@ testlogger {
     showPassed false
     showSkipped false
     showFailed true
+    showOnlySlow false
 }
 ```
 
-By default all the above three flags are turned on. If you have chosen to display standard streams by setting
+By default all the above four flags are turned on. If you have chosen to display standard streams by setting
```

Comment:
`showOnlySlow` is will be off by default. So this goes back to being

> By default the flags `showPassed`, `showSkipped` and `showFailed` are turned on while `showOnlySlow` will be off.

Reply:
done ;) just inserted your quote.